crates/cubecl-wgpu-spirv/Cargo.toml

```diff
@@ -23,13 +23,13 @@ cubecl-spirv = { path = "../cubecl-spirv", version = "0.2.0" }
 cubecl-wgpu = { path = "../cubecl-wgpu", version = "0.2.0" }
 
 bytemuck = { workspace = true }
-pollster = { workspace = true }
 wgpu = { version = "22.0.0", features = [
     "fragile-send-sync-non-atomic-wasm",
     "spirv",
 ] }
 
 async-channel = { workspace = true }
+futures-lite = { workspace = true }
```

Comment:
What is the difference between `pollster` and `futures-lite`?

Reply:
Ah yes forgot this in the PR description - we now need futures-lite for some extension methods anyway (mainly the futures panic unwinding), and it has a block_on method much like pollster. Think it's a bit more actively supported and don't need two deps.